src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/AspNetTelemetryEventSource.cs

```diff
@@ -37,7 +37,21 @@ public void ActivityStarted(Activity activity)
         {
             if (this.IsEnabled(EventLevel.Verbose, EventKeywords.All))
             {
-                this.ActivityStarted(activity?.Id);
+                // Accessing activity.Id here will cause the Id to be calculated and cached
```

Comment:
if SDK eventsource is already logging this, do we need it again from the asp.net as well?

Reply:
you are right - This one simply logs if activity was started or not based on sampling result. I will remove this one.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/69ef5f86bf6ef2cf8b36a8008050a3f7fcd47f35/src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs#L81